crypto/BRCryptoWallet.c

```diff
@@ -681,20 +681,18 @@ cryptoWalletEstimateFeeBasis (BRCryptoWallet  wallet,
         }
 
         case BLOCK_CHAIN_TYPE_GEN: {
-            //            BRGenericWalletManager gwm = wallet->u.gen.gwm;
-            //            BRGenericWallet wid = wallet->u.gen.wid;
-            //
-            //            UInt256 genValue = cryptoAmountGetValue(amount);
-
-            // TODO: Generic EstimateFee
-            assert (0);
```

Comment:
Get rid of assert for now so that the submit flow can work.

Reply:
One other thing.  I have another local change to Swift/BRCryptoDemo/TransferCreateSendController.swift that is not part of this change but is required for the demo: a default BRCryptoFeeBasis object since we don't have a callback to create one yet.  Should I check that in?